### PR TITLE
refactor(es_extended): Improve optimization

### PR DESCRIPTION
**Pull request by Tinky124**

### CODE REVIEW

**Review by JackDUpModZ — COMMENTED** [review]
These changes are unrequired and can be added simply with a few lines of code in a client lua file, Making these changes into the framework would prevent people from using them and cause conflict

**Review by Tinky124 — COMMENTED** [review]
Update

File: [esx]/es_extended/client/main.lua [hunk 1]
```diff
@@ -64,53 +64,58 @@ AddEventHandler('esx:playerLoaded', function(xPlayer, isNew, skin)
 			local RemoveAllPickupsOfType = RemoveAllPickupsOfType
 			local HideHudComponentThisFrame = HideHudComponentThisFrame
 			local PlayerId = PlayerId()
+			local ESXPlayerId = ESX.PlayerData.ped
```
Comment (Gittified) [inline [esx]/es_extended/client/main.lua:67]: Why is this called 'ESXPlayerId' and not 'playerPed' or something similar?

File: [esx]/es_extended/client/main.lua [hunk 2]
```diff
@@ -64,53 +64,58 @@ AddEventHandler('esx:playerLoaded', function(xPlayer, isNew, skin)
...
 						SetPlayerLockonRangeOverride(PlayerId, 2.0)
 					end
 				end
 
 				if Config.DisableVehicleRewards then
-					Sleep = false
-					DisablePlayerVehicleRewards(PlayerId)
+					local veh = GetVehiclePedIsEntering(ESXPlayerId)
+        			if veh ~= 0 then
+						Sleep = false
```
Comment (Gittified) [inline [esx]/es_extended/client/main.lua:103]: This does not do anything except for delay the next call

**Review by Gittified — CHANGES_REQUESTED** [review]
Looking good, please take a look at the comments :)

File: [esx]/es_extended/client/main.lua (hunk 1 above)
Comment (Tinky124) [reply [esx]/es_extended/client/main.lua]: Fixed @Gittified

File: [esx]/es_extended/client/main.lua (hunk 2 above)
Comment (Tinky124) [reply [esx]/es_extended/client/main.lua]: Removed delay @Gittified

**Review by Gittified — APPROVED** [review]
Looks good to me

**Comment by Tinky124** [comment]
Since this community is toxic i dont care anymore for esx, and i dont want to use it anymore qbus is better.

**Comment by Benzo00** [comment]
> Since this community is toxic i dont care anymore for esx, and i dont want to use it anymore qbus is better.

This Comment but still reopened the pr? 🤔

**Comment by Tinky124** [comment]
@Benzo00 If you want merge..

**Comment by Gellipapa** [comment]
@Tinky124 I believe that ESX is better than qbus, but thanks a lot for these fixes.

**Comment by Gellipapa** [comment]
It looks good what to do with it  @Benzo00?

**Review by TheFantomas — APPROVED** [review]
Whats up. What are you waiting for? :D

**Comment by Rav3n95** [comment]
> Whats up. What are you waiting for? :D

Benzo

**Comment by Gellipapa** [comment]
@Tinky124 Hi! Please update your branch and fix merge conflicts after we can merge. Thank you for help :)

**Review by Benzo00 — APPROVED** [review]

**Comment by CsokiHUN** [comment]
I resolved the conflicts, check if it works well. @Benzo00 @Gellipapa

**Comment by Gellipapa** [comment]
> 

Thank you for help @Csoki i will tested coming soon.

**Pull request closed without merging**